src/app/services/data.service.ts

```diff
@@ -116,6 +186,18 @@ export class DataService {
     return Object.values(this.data.vehicles);
   }
 
+  public getActiveVehicles(): Vehicle[] {
+    return Object.values(this.data.vehicles).filter((v) => v.active);
+  }
+
+  public getDefaultVehicles(): Vehicle[] {
+    return Object.values(this.data.vehicles).filter((v) => v.default);
+  }
+
+  public getCustomActiveVehicles(): Vehicle[] {
+    return Object.values(this.data.vehicles).filter((v) => !v.default && v.active);
+  }
```

Comment:
getDefaultVehicles() and getCustomActiveVehicles() not used anymore

Reply:
should be used by the vehicle component - so I'd keep those functions

Reply:
true